package.json

```diff
@@ -59,6 +59,8 @@
     "ember-resolver": "^4.3.0",
     "ember-sinon": "^1.0.0",
     "ember-source": "~2.16.0",
+    "fastboot-transform": "^0.1.2",
+    "fs-extra": "^4.0.0",
```

Comment:
Why do we need this? I don’t see it being used here in this PR?

Reply:
Sorry this was a merge conflict issue.